cmd/util.go

```diff
@@ -14,26 +16,58 @@ var (
 	CommandPrefix string
 )
 
+// flagConfig compares command line flags and the flags set in the config
+// files.  The command line value will always override any value set in the
+// config files.
+func flagConfig(fs *flag.FlagSet) {
+	fs.VisitAll(func(f *flag.Flag) {
+		var (
+			configValue  interface{}
+			configString string
+		)
+
+		switch f.Value.Type() {
+		case "bool":
+			configValue = getMainConfig().GetBool(CommandPrefix + f.Name)
+			configString = strconv.FormatBool(configValue.(bool))
+		case "string":
+			configValue = getMainConfig().GetString(CommandPrefix + f.Name)
+			configString = configValue.(string)
+		case "stringSlice":
```

Comment:
may need to include support for `stringArray` (not sure if it's a unique type under the hood...), per https://github.com/zaquestion/lab/pull/431

Reply:
I think I'm okay because I have a log.Fatal() for the default case.  IOW code goes BOOM if there's an unidentified type.

Reply:
So LOL after a rebase the code did go boom.  The problem is that viper doesn't have support for stringArray.  For now I've put a comment in the code that stringArray is unsupported in viper.  Given how stringArray is being used in the code with -m on note commands, I doubt anyone will want to set it in the config files.  It's something we can fix later.